gradle/runBootMicroserviceAcceptanceTests.sh

```diff
@@ -16,10 +16,11 @@ dumpCurrentMicroInfraSpringVersionToFile
 
 git clone https://github.com/4finance/boot-microservice.git
 cd boot-microservice
+git checkout tech/integration_with_sleuth
 updateMicroInfraSpringVersionInConfigurationFile
 ./gradlew clean check uptodate --stacktrace --continue --info --no-daemon
 
 git reset --hard
-git checkout boot-microservice-gui
+git checkout tech/integration_with_sleuth_gui
```

Comment:
Please remember to restore the original branch when merged in boot-microservice.